### PR TITLE
[PM-6567] migrate toast to CL service

### PR DESCRIPTION
**Pull request by jaasen-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-6567

## 📔 Objective

This PR migrates all use of the deprecated `PlatformUtilsService.showToast` to instead use the `toastService`.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/desktop/src/auth/login/login.component.ts [hunk 1]
```diff
@@ -161,7 +164,7 @@ export class LoginComponent extends BaseLoginComponent implements OnInit, OnDest
   async continue() {
     await super.validateEmail();
     if (!this.formGroup.controls.email.valid) {
-      this.platformUtilsService.showToast(
+      this.toastService.showToast(
         "error",
         this.i18nService.t("errorOccured"),
         this.i18nService.t("invalidEmail"),
```
Comment (rr-bw) [inline apps/desktop/src/auth/login/login.component.ts:170]: Just missing the object and the `variant`, `title`, and `message` keys.

**Review by rr-bw — CHANGES_REQUESTED** [review]

File: apps/desktop/src/auth/login/login.component.ts (hunk 1 above)
Comment (jaasen-livefront) [reply apps/desktop/src/auth/login/login.component.ts]: @rr-bw nice catch! fixed.

**Review by willmartian — CHANGES_REQUESTED** [review]
Hey Jordan! 

1. `libs/common` cannot have dependencies on Angular (which includes the `ToastService` and `libs/components` generally). Same restriction also applies to `libs/auth/common`. All changes in these areas should be reverted. 

Those areas should either be refactored to happen elsewhere (in an Angular context), or the CL team needs to expose a non-Angular implementation that uses message passing (similar to the `PlatformUtilsService.showToast`). That doesn't need to happen in this PR.

**Comment by jaasen-livefront** [comment]
> Hey Jordan!
> 
> `libs/common` cannot have dependencies on Angular (which includes the `ToastService` and `libs/components` generally). Same restriction also applies to `libs/auth/common`. All changes in these areas should be reverted.
> 
> Those areas should either be refactored to happen elsewhere (in an Angular context), or the CL team needs to expose a non-Angular implementation that uses message passing (similar to the `PlatformUtilsService.showToast`). That doesn't need to happen in this PR.

Understood!

**Pull request closed without merging**